pkgs/development/python-modules/pycocotools/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, cython
+, fetchPypi
+, lib
+, matplotlib
+, numpy
+, pytest
+}:
+buildPythonPackage rec {
+  pname = "pycocotools";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version ;
+    sha256 = "cbb8c2fbab80450a67ee9879c63b0bc8a69e58dd9a0153d55de404c0d383a94b";
+  };
+
+  nativeBuildInputs = [
+    cython
+    numpy
```

numpy will likely be needed at runtime
```suggestion
  ];

  propagatedBuildInputs = [  numpy
```